tests/scripts/thread-cert/node.py

```diff
@@ -67,13 +74,21 @@ def __init__(self, nodeid, is_mtd=False, simulator=None):
 
     def __init_sim(self, nodeid, mode):
         """ Initialize a simulation node. """
+
         if 'OT_CLI_PATH' in os.environ.keys():
             cmd = os.environ['OT_CLI_PATH']
-        elif 'top_builddir' in os.environ.keys():
+        elif (self.version == '1.1' and self.version != self.env_version):
+            # rcp posix app
+            if 'OT_CLI_PATH_1_1' in os.environ.keys():
```

Comment:
```suggestion
            if 'OT_CLI_PATH_1_1' in os.environ:
```

Reply:
@simonlingoogle 
it is a little complex as this part is distributed in both #4325 and this PR. it could work so far, we can keep it a is and update in a unified PR afterwards?

/cc/ @bukepo